corehq/apps/geospatial/templates/map_visualization.html

```diff
@@ -84,6 +84,16 @@
     <a id="btnSaveDrawnArea" class="col-sm-2 btn btn-default" style="float:right; margin-right:1em" data-bind="attr: { disabled: btnSaveDisabled }">
         {% trans 'Save Area' %}
     </a>
+    <a id="btnRunDisbursement" class="col-sm-2 btn btn-primary" style="float:right; margin-right:1em" data-bind="attr: { disabled: btnRunDisbursementDisabled }">
+        {% trans 'Run Disbursement' %}
+    </a>
+</div>
+<div id="disbursement-spinner">
+  <h4 id="loading" class="hide"
+      data-bind="visible: isBusy(), css: {hide: false}">
+    <i class="fa fa-spin fa-spinner"></i>
+    {% trans "Running disbursement algorithm..." %}
```

Comment:
This is completely fine as is, but I wonder if we should display what disbursement algorithm is running? Something like "Running 'radial' disbursement algorithm...". What are your thoughts @Charl1996?